grpc/build.gradle

```diff
@@ -30,11 +30,9 @@ sourceSets {
         proto {
             srcDirs 'src/main/proto'
```

Comment:
Please, correct the project's structure as it was done in https://github.com/th2-net/th2-read-db/tree/dev/grpc